src/api/c/lu.cpp

```diff
@@ -49,6 +49,9 @@ af_err af_lu(af_array *lower, af_array *upper, af_array *pivot,
 
         af_dtype type = i_info.getType();
 
+        ARG_ASSERT(0, lower != nullptr);
```

Comment:
Are we going to add such `nullptr` checks for all functions going forward ?

Reply:
We should but not now.